tarodb/assets_store_test.go

```diff
@@ -867,6 +874,18 @@ func TestAssetExportLog(t *testing.T) {
 	require.NoError(t, err)
 	require.Equal(t, 1, len(assetTransfers))
 
+	// Check that the new UTXO is found among our managed UTXOs.
+	utxos, err = assetsStore.FetchManagedUTXOs(ctx)
+	require.NoError(t, err)
+	require.Len(t, utxos, 2)
+	require.Equal(t, assetGen.anchorPoints[0], utxos[0].OutPoint)
+	require.Equal(t, spendDelta.NewAnchorPoint, utxos[1].OutPoint)
```

Comment:
nit: I would consider adding 
```go
newUtxo := utxo[1]
```
after line 880. And then using `newUtxo` for the remaining assertions.

Reply:
Good idea. Done: 43f0ef237215e2b6cb691c5d14159a96a6027dd3